src/app/pages/classes-page/classes-page.component.scss

```diff
@@ -0,0 +1,222 @@
+@use "../../../import" as *;
+
+.circle{
+    height: 30px;
+    width: 30px;
+    border-radius: 30px;
+    background-color:#DDB375;
+    color: black;
+    display:flex;
+    align-items: center;
+    justify-content:center;
+}
+
+.text-description{
+    margin-left:5%;
```

Comment:
Minify this declaration with margin: 2% 5% 0 5% => 
clockwise direction =>  margin: top right bottom left;